popup.html

```diff
@@ -135,14 +136,16 @@
             type="button"
             id="scanAndRandomize"
             class="btn btn-outline-danger btn-sm"
+			style="background-color: #0074D9; color: #FFFFFF;"
           >
-            SCAN & RANDOMIZE
+            SCAN & RAND
```

Comment:
Some people might not understand the abbreviation, let's stick with the old version.

Reply:
Thank you for the review. I'll stick with the old style and colors. and I'll update the story and votes element according to your feedback.